src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -477,6 +532,7 @@ private static bool RemapRuleDefaultLevelFromOpenToNote(JObject resources)
                     // reasonable level to associate with this class of report, if it is emitted. In 
                     // practice, we don't expect that a current producer exists who is in this condition.
                     configuration["defaultLevel"] = "note";
+                    modifiedResources = true;
```

Comment:
>modifiedResources [](start = 20, length = 17)

*Biting my tongue* ;-)

Reply:
Prudent. And still your point is made. :) We'll get this fixed.

---
In reply to: [249162790](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249162790) [](ancestors = 249162790)

Reply:
Fixed later, I mean. :)

---
In reply to: [249219447](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249219447) [](ancestors = 249219447,249162790)